python/ray/train/_internal/state/schema.py

```diff
@@ -46,9 +63,19 @@ class TrainRunInfo(BaseModel):
     datasets: List[TrainDatasetInfo] = Field(
         description="A List of dataset info for this Train run."
     )
+    run_status: RunStatusEnum = Field(
+        description="The current status of the train run. It can be one of the "
+        "following: STARTED, FINISHED, or ERRORED."
+    )
     start_time_ms: int = Field(
         description="The UNIX timestamp of the start time of this Train run."
     )
+    end_time_ms: Optional[int] = Field(
+        description="The UNIX timestamp of the end time of this Train run."
+    )
+    controller_actor_status: Optional[ActorStatusEnum] = Field(
```

Comment:
how should this field be used? We should show it as a column to the user? What happens if the controller actor dies?

Reply:
Currently if the controller actor dies, the train run will crash.

Probably we don't need to expose this internal details, and only shows the `run_status`?

Reply:
yeah, lets leave it out for now.